stumpy/scrump.py

```diff
@@ -472,46 +568,90 @@ def scrump(T, m, percentage=0.01, pre_scrump=False, s=None):
     See Algorithm 1 and Algorithm 2
     """
 
-    T = np.asarray(T)
-    T = T.copy()
-    T[np.isinf(T)] = np.nan
-    core.check_dtype(T)
+    if T_B is not None and pre_scrump:  # pragma: no cover
+        raise NotImplementedError(
+            "AB joins with preprocessing are not yet implemented."
+        )
```

Comment:
What would we need to do to implement this? Should we start a new issue to cover this?

Reply:
For this we would need to adapt prescrimp to work on AB joins.

Reply:
Do you need any help with this?